source/skins/simple/OptionsSkin.js

```diff
@@ -114,17 +128,58 @@ export const OptionsSkin = (props: Props) => {
       // call user's custom rendering logic
       return optionRenderer(option);
     } else if (isObject(option)) {
-      return <span className={theme[themeId].label}>{option.label}</span>;
+      let { label } = option;
+      // in case `highlightSearch` then `searchValue` is wrapped in an `em` tag
+      if (highlightSearch !== false) {
+        const splitter = new RegExp(`(${searchValue})`,'i');
+        const parts = label.split(splitter);
+        for (let i = 1; i < parts.length; i += 2) {
+          if (parts[i].toLowerCase() === searchValue.toLowerCase())
+            parts[i] = <em>{parts[i]}</em>;
+          label = parts;
+        }
+      }
+      return <span className={theme[themeId].label}>{label}</span>;
     }
     return option;
   };
 
+  const renderSearch = () => {
+    return (
+      <div className={theme[themeId].search}>
+        <Input
+          skin={InputSkin}
```

Comment:
Can you try to remove the direct skin reference + prop - this shouldn't be necessary as the skins should be injected via context (otherwise we cannot override them if necessary)

Reply:
I've removed only this instance. Did you mean through the whole app?

Reply:
It's fine only for this PR, we need to rewrite the whole library in Typescript at some point anyway 😆